script/windows-installer/inno-setup-git-lfs-installer.iss

```diff
@@ -1,9 +1,9 @@
-; Script generated by the Inno Script Studio Wizard.
-; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
-
 #define MyAppName "Git LFS"
-#define MyAppVersion "1.5.0"
-#define MyAppPublisher "GitHub, Inc"
+
+; Misuse RemoveFileExt to strip the 4th patch-level version number.
+#define MyAppVersion RemoveFileExt(GetFileVersion("..\..\git-lfs-x86.exe"))
```

Comment:
Nope, I arbitrarily chose the x86 version. As both executables have the version information embedded it does not matter which one Inno Setup is looking at here. But I should make that clearer by adding a comment.